javascript/big-practice/src/scripts/controllers/noteController.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @class noteController
+ * @description Controller is an intermediary for views and models
+ *
+ * @param model
+ * @param view
+ */
+export default class NoteController {
+  constructor(model, view) {
+    this.model = model;
+    this.view = view;
+  }
+
+  init() {
+    this.renderAllNotes();
+    this.view.bindShowAndAddInput(this.addNote);
+  }
+
+  /**
+   * @description render all the notes
+   */
+  renderAllNotes = () => {
+    this.view.constructor.renderListNotes(this.model.notes);
+  };
+
+  /**
+   * @description function add note
+   *
+   * @param {String} title is title from input
+   * @param {String} description is description from input
+   */
+  addNote = (title, description) => {
+    const note = this.model.addNoteModel(title, description);
+    this.view.constructor.deleteAllNotes(note);
```

Comment:
In `addNote` call `deleteAllNotes` does not make sense to me. 

Could you explain?